docs/content/routing/providers/docker.md

```diff
@@ -171,6 +174,22 @@ and the router automatically gets a rule defined by `defaultRule` (if no rule fo
       - "traefik.http.routers.myproxy.rule=Host(`example.net`)"
     ```
 
+??? example "Explicit definition with multiple Services"
```

Comment:
Is this example about port selection?

Reply:
Nope. It's about showing explicit definition with one service. I tried to fix it with 42d28fd08.